app/src/main/java/com/sun/heartrate/ui/history/HistoryFragment.kt

```diff
@@ -27,29 +45,130 @@ class HistoryFragment : Fragment(),
     
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+        initListener()
         initAdapter()
+        setHeartRateLastView()
+    }
+    
+    private fun setHeartRateLastView() {
+        heartModelLast = hearts[(hearts.size) - 1]
+        textViewNumberRate?.text = heartModelLast.heartRate.toString()
+        textViewNote?.text = heartModelLast.title
+        textViewDate?.text = heartModelLast.time.formatDate()
+        imageStatus?.setImageResource(heartModelLast.image)
+    }
+    
+    private fun initListener() {
+        assignViews(imageViewDelete)
     }
     
     override fun menuOptionCallBack(value: String) {
-        // TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
+        showViewMenuOption(
+            value,
+            recyclerMonth,
+            recyclerHeartDetail,
+            includePartialFragmentHistory
+        )
+    }
+    
+    private fun showViewMenuOption(
```

Comment:
Em thấy hàm này không được xịn xò lắm, anh có thể dùng "vararg views: View?" giống như cách anh cài đặt assignViews
```suggestion
    private fun showViewMenuOption(option: String, vararg views: View?){
```